package.json

```diff
@@ -1,21 +1,18 @@
 {
   "name": "@modusoperandi/licit-glossary",
-  "version": "0.0.3",
+  "version": "0.0.4",
+  "type": "module",
   "description": "MO's Glossary ProseMirror Plugin",
   "main": "dist/index.js",
-  "typings": "./index.d.ts",
+  "types": "dist/index.d.ts",
   "license": "UNLICENSED",
   "repository": {
     "type": "git",
     "url": ""
   },
-  "maintainers": [
-    {
-      "name": "Firmusoft Solutions"
-    }
-  ],
   "dependencies": {
-    "@modusoperandi/licit-ui-commands": "^0.1.15",
+    "@modusoperandi/licit-doc-attrs-step": "^0.1.6",
```

Comment:
These should be peerDependencies. And please confirm if prosemirror-example-setup should be a devDependency.